src/Autodesk.Forge.DesignAutomation/ApiClient.cs

```diff
@@ -46,8 +44,15 @@ public async Task UploadAppBundleBits(UploadAppBundleParameters uploadParameters
                 using (var content = new StreamContent(new FileStream(packagePath, FileMode.Open)))
                 {
                     formData.Add(content, "file");
-                    var response = await Service.Client.PostAsync(uploadParameters.EndpointURL, formData);
-                    response.EnsureSuccessStatusCode();
+
+                    using (var request = new HttpRequestMessage(HttpMethod.Post, uploadParameters.EndpointURL) { Content = formData })
+                    {
+                        // increase timeout to 10 minutes
+                        request.Properties.Add(ForgeConfiguration.TimeoutKey, 10 * 60);
```

Comment:
I was thinking to add the appbundle upload timeout value to configuration, but reference to the configuration is not stored in `DesignAutomationClient` ctor (which is auto-generated), so I cannot access it from here.
Hardcoded for now

Let me know if codegen should be changed, I could take a look.

Reply:
Yeah, we should store a reference to the Configuration. We already pass it in we just don't store it.